pyfiglet/__init__.py

```diff
@@ -336,10 +340,11 @@ def __char(data):
             # Load German Umlaute - the follow directly after standard character 127
             if data:
```

Comment:
Maybe drop this `if` then?

Reply:
Oops.  That was an attempt to fix gradient, which is actually a broken file...  Some definitions are missing lines and so pyfiglet runs out of data on the last character.  I originally thought it was missing the enter definition, but it dies part-way through in `__char()`.

The real question should probably be whether pyfiglet should soldier on, or reject the font entirely.

Reply:
My assumption, when I added that was that fonts would either have all these german umlaut extensions or not. Just a wild guess, I don't know how the fonts are usually structured.

Reply:
Yeah - that's what figlet does too, but it's clearly happy to stop processing mid character and just use what it found.

Reply:
What's the conclusion here. Ready to go or is something outstanding here? I don't have a strong opinion, though possibly it's better to follow figlet's behaviour?

Reply:
I'll keep it simple and just remove the erroneous check.